app/controllers/voucher_adjustments_controller.rb

```diff
@@ -6,8 +6,9 @@ class VoucherAdjustmentsController < BaseController
   def create
     if add_voucher
       VoucherAdjustmentsService.calculate(@order)
+      @order.update_totals_and_states
 
-      render_voucher_section
+      update_payment_section
```

Comment:
We're updating the payment section 👍. But no longer updating the voucher section?

Reply:
Oh, I that's brought back later in another commit.